system/include/compat/avxintrin.h

```diff
@@ -44,215 +535,1271 @@ _mm_broadcast_ss(const float *__mem_addr)
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testnzc_pd(__m256d __a,
+                                                         __m256d __b) {
+  v128_t __m =
+    wasm_u64x2_shr(wasm_v128_and((v128_t)__a.v0, (v128_t)__b.v0), 63);
+  v128_t __m1 =
+    wasm_u64x2_shr(wasm_v128_and((v128_t)__a.v1, (v128_t)__b.v1), 63);
+  v128_t __m2 =
+    wasm_u64x2_shr(wasm_v128_andnot((v128_t)__b.v0, (v128_t)__a.v0), 63);
+  v128_t __m3 =
+    wasm_u64x2_shr(wasm_v128_andnot((v128_t)__b.v1, (v128_t)__a.v1), 63);
+  return wasm_v128_any_true(wasm_v128_or(__m, __m1)) &
+         wasm_v128_any_true(wasm_v128_or(__m2, __m3));
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testz_ps(__m256 __a, __m256 __b) {
+  return _mm_testz_ps(__a.v0, __b.v0) & _mm_testz_ps(__a.v1, __b.v1);
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testc_ps(__m256 __a, __m256 __b) {
+  return _mm_testc_ps(__a.v0, __b.v0) & _mm_testc_ps(__a.v1, __b.v1);
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testnzc_ps(__m256 __a,
+                                                         __m256 __b) {
+  v128_t __m =
+    wasm_u32x4_shr(wasm_v128_and((v128_t)__a.v0, (v128_t)__b.v0), 31);
+  v128_t __m1 =
+    wasm_u32x4_shr(wasm_v128_and((v128_t)__a.v1, (v128_t)__b.v1), 31);
+  v128_t __m2 =
+    wasm_u32x4_shr(wasm_v128_andnot((v128_t)__b.v0, (v128_t)__a.v0), 31);
+  v128_t __m3 =
+    wasm_u32x4_shr(wasm_v128_andnot((v128_t)__b.v1, (v128_t)__a.v1), 31);
+
+  return wasm_v128_any_true(wasm_v128_or(__m, __m1)) &
+         wasm_v128_any_true(wasm_v128_or(__m2, __m3));
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testz_si256(__m256i __a,
+                                                          __m256i __b) {
+  return _mm_testz_si128(__a.v0, __b.v0) & _mm_testz_si128(__a.v1, __b.v1);
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testc_si256(__m256i __a,
+                                                          __m256i __b) {
+  return _mm_testc_si128(__a.v0, __b.v0) & _mm_testc_si128(__a.v1, __b.v1);
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_testnzc_si256(__m256i __a,
+                                                            __m256i __b) {
+  v128_t __m = wasm_v128_and(__a.v0, __b.v0);
+  v128_t __m1 = wasm_v128_and(__a.v1, __b.v1);
+  v128_t __m2 = wasm_v128_andnot(__b.v0, __a.v0);
+  v128_t __m3 = wasm_v128_andnot(__b.v1, __a.v1);
+  return wasm_v128_any_true(wasm_v128_or(__m, __m1)) &
+         wasm_v128_any_true(wasm_v128_or(__m2, __m3));
+}
+
+static __inline int __DEFAULT_FN_ATTRS _mm256_movemask_pd(__m256d __a) {
+  return _mm_movemask_pd(__a.v0) | (_mm_movemask_pd(__a.v1) << 2);
 }
 
+static __inline int __DEFAULT_FN_ATTRS _mm256_movemask_ps(__m256 __a) {
+  return _mm_movemask_ps(__a.v0) | (_mm_movemask_ps(__a.v1) << 4);
+}
+
+static __inline void __DEFAULT_FN_ATTRS _mm256_zeroall(void) {
+  UNIMPLEMENTED("_mm256_zeroall");
+}
+
+static __inline void __DEFAULT_FN_ATTRS _mm256_zeroupper(void) {
+  UNIMPLEMENTED("_mm256_zeroupper");
```

Comment:
I think the above two functions can safely be empty functions, there shouldn't be any drawbacks since when porting any assembly code that would have calls to these functions around, that assembly code in the first place will not compile.

After developer removes all uses and calls to manual x86 assembly code, the lack of action in zeroall and zeroupper functions will not be observable, and can safely do nothing.